tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +147,18 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert len(jesse.inventory) == len(tai.inventory)
+    assert item_c not in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_f not in jesse.inventory
+    assert item_f in tai.inventory
+    assert item_a in tai.inventory
+    assert item_a not in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_d not in tai.inventory
+    assert result
```

Comment:
See comment above about explicitly testing for the value of `result` instead of checking that `result` is a truthy value.